ATPDocs/troubleshooting-known-issues.md

```diff
@@ -9,6 +9,66 @@ ms.topic: troubleshooting
 
 This article describes how to troubleshoot known issues in Microsoft Defender for Identity. 
 
+## Sensor service fails to start
+
+**Sensor log entries:**
+
+`Warn DirectoryServicesClient CreateLdapConnectionAsync failed to retrieve group managed service account password. [DomainControllerDnsName=DC1.CONTOSO.LOCAL Domain=contoso.local UserName=AATP_gMSA]`
+
+### Cause 1
+
+The domain controller hasn't been given rights to access the password of the gMSA account.
+
+**Resolution 1:**
+
+Verify that the domain controller has been given rights to access the password. You should have a Security Group in Active Directory that contains the domain controller(s), AD FS / AD CS server(s) and standalone sensors computer accounts included. If this doesn't exist, we recommend that you create one.
+
+You can use the following command to check if a computer account or security group has been added to the parameter. Replace *mdiSvc01* with the name you created.
+
+```powershell
+Get-ADServiceAccount mdiSvc01 -Properties PrincipalsAllowedToRetrieveManagedPassword
+```
+
+The results should look like this:
+
+![Powershell results.](media/troubleshooting-known-issues/gmsa-retrieve-password-results.png)
+
+In this example, we can see that a group named *mdiSvc01Group* has been added. If the domain controller or the security group hasn't been added, you can use the following commands to add it. Replace *mdiSvc01* with the name of gMSA, and replace *DC1* with the name of the domain controller, or *mdiSvc01Group* with the name of the security group.
+
+```powershell
+# To set the specific domain controller only:
+$specificDC = Get-ADComputer -Identity DC1
+Set-ADServiceAccount mdiSvc01 -PrincipalsAllowedToRetrieveManagedPassword $specificDC
+
+
+# To set a security group that contains the relevant computer accounts:
+$group = Get-ADGroup -Identity mdiSvc01Group
+Set-ADServiceAccount mdiSvc01 -PrincipalsAllowedToRetrieveManagedPassword $group
+```
+
+If the domain controller or security group is already added, but you're still seeing the error, you can try the following steps:
+
+- **Option 1**: Reboot the server to sync the recent changes
+- **Option 2**:
+    1. Set the **AATPSensor** and **AATPSensorUpdater** services to Disabled
+    1. Stop the **AATPSensor** and **AATPSensorUpdater** services
+    1. Cache service account to server using the command: `Install-ADServiceAccount gMSA_AccountName`
```

Comment:
@anunesms 
Let's remove option 2 from here. 
There's no need to install the service account on the server. It's relevant only for MSAs and not gMSAs.